test/InterchainToken.js

```diff
@@ -147,7 +147,7 @@ describe('InterchainToken', () => {
             const contractBytecodeHash = keccak256(contractBytecode);
 
             const expected = {
-                london: '0x482146829055f052063003e9cf0ffaf798a12fb58088c2667566a135b9568355',
+                london: '0xa01cf28b0b6ce6dc3b466e995585d69486400d671fce0ea8d06beba583e6f3bb',
```

Comment:
prevent changing this bytecode since it's not necessary

Reply:
this is changed back to the original bytecode